src/liquidity_pool_call_builder.ts

```diff
@@ -36,4 +36,24 @@ export class LiquidityPoolCallBuilder extends CallBuilder<
     this.url.setQuery("reserves", commaSeparatedAssets);
     return this;
   }
+
+  /**
+   * Retrieves a specific liquidity pool by ID.
+   *
+   * @param  {string} id
+   * @returns {CallBuilder} a new CallBuilder instance for the /liquidity_pools/:id endpoint
+   */
+  public liquidityPoolId(
+    id: string,
+  ): CallBuilder<ServerApi.LiquidityPoolRecord> {
+    if (!id.match(/[a-fA-F0-9]{64}/)) {
+      throw new Error(`${id} does not look like a liquidity pool ID`);
+    }
```

Comment:
I like that you're adding this validation 🙌 . Could you add a test to validate this?

Reply:
There is one, [here](https://github.com/stellar/js-stellar-sdk/pull/687/files#diff-fdc571fda86e588a983c87a62cba0c947cf4193ad9263c22b6af766b6d322cfdR181-R183)!

Reply:
Also expanded in 848d054.